datahub-web-react/src/app/shared/ManageAccount.tsx

```diff
@@ -16,17 +18,43 @@ const defaultProps = {
 
 export const ManageAccount = ({ urn: _urn, pictureLink: _pictureLink }: Props) => {
     const entityRegistry = useEntityRegistry();
+
+    const isLoggedIn = checkAuthStatus();
+
+    const handleLogout = () => {
+        Cookies.remove('PLAY_SESSION');
+        Cookies.remove('IS_LOGGED_IN');
+        localStorage.removeItem('userUrn');
+        isLoggedInVar(false);
+    };
+
+    const menu = (
+        <Menu>
+            <Menu.Item danger>
+                <div tabIndex={0} role="button" onClick={handleLogout} onKeyDown={handleLogout}>
+                    Log out
+                </div>
+            </Menu.Item>
+        </Menu>
+    );
+
+    if (!isLoggedIn) {
+        return <Redirect to="/login" />;
+    }
```

Comment:
do you need this redirect? Each page is wrapped in a ProtectedRoute component which should be doing this redirect for you

Reply:
You are right, this is removed now.